### PR TITLE
Fix GValue size on i686-pc-windows-gnu

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gtk/issues/434

### CODE REVIEW

**Comment by EPashkin** [comment]
All not stable CI passed, stables still have `failed lock source` error.

**Comment by EPashkin** [comment]
@GuillaumeGomez May be we need also add this compile check to prevent `GValue` fail on big pointers?
```
#[cfg(target_pointer_width = "128")]
const ERROR: () = "Your pointers are too big.";
```

**Comment by GuillaumeGomez** [comment]
I was thinking about it as well. Can it be done directly in gir by any chance?

**Comment by EPashkin** [comment]
Yes, it can be inserted before `struct GValue` (only needed "if" in `generate_record`)

**Comment by GuillaumeGomez** [comment]
Can you do it please? :D

**Comment by EPashkin** [comment]
Build test

**Comment by EPashkin** [comment]
Added pointer size check

File: gobject-sys/src/lib.rs [hunk 1]
```diff
@@ -417,10 +417,13 @@ pub struct GTypeValueTable {
     pub lcopy_value: Option<unsafe extern "C" fn(*const GValue, c_uint, *mut GTypeCValue, c_uint) -> *mut c_char>,
 }
 
+#[cfg(target_pointer_width = "128")]
```
Comment (GuillaumeGomez) [inline gobject-sys/src/lib.rs]: Awesome!

File: gobject-sys/src/lib.rs [hunk 2]
```diff
@@ -417,10 +417,13 @@ pub struct GTypeValueTable {
...
+const ERROR: () = "Your pointers are too big.";
+
 #[repr(C)]
 pub struct GValue {
     g_type: GType,
-    pub data: [gpointer; 2],
```
Comment (GuillaumeGomez) [inline gobject-sys/src/lib.rs]: Won't it be a problem on 32 bits targets?

Comment (EPashkin) [inline gobject-sys/src/lib.rs]: See https://github.com/gtk-rs/gtk/issues/434. It actualy fixed problem :wink:

Comment (GuillaumeGomez) [inline gobject-sys/src/lib.rs]: So weird...

**Comment by GuillaumeGomez** [comment]
Appveyor passed, strange. Is it ready for you?

**Comment by EPashkin** [comment]
Yes, all ready. Appveyour passed as it used old rust v1.9.0

**Comment by GuillaumeGomez** [comment]
It explains things... Well, thanks a lot for this fix!

**Pull request merged**